src/cache/shared.ts

```diff
@@ -68,6 +68,7 @@ export type CacheEntry = {
   scope: string;
   client_id: string;
   refresh_token?: string;
+  oauthTokenScope?: string;
```

Comment:
As mentioned in the description, this is an attempt to "remember" the actual scopes that came back from `/oauth/token` (if any) so that they can be reliably returned in `getTokenSilently` if the verbose response is asked for, mainly for consistency.